README.md

```diff
@@ -105,6 +107,10 @@ $ curl -F "image=@assets/surfing.jpg" -X POST http://127.0.0.1:5000/model/predic
 
 To run the Flask API app in debug mode, edit `config.py` to set `DEBUG = True` under the application settings. You will then need to rebuild the docker image (see [step 1](#1-build-the-model)).
 
+## 5. Cleanup
```

Comment:
The link on line 37 goes to `clean-up` but the HTML anchor generated by this title is `cleanup`

Reply:
Good catch thanks - updated